make test fails with conversion errors

```
27fb29696b31:/build/talek# make test
go fmt ./...
cli/clinfo/main.go
cli/talekutil/main.go
common/trustdomain_config.go
cuckoo/table.go
libtalek/handle.go
pir/pircl.go
pir/pircuda.go
pir/pircl/context_cl.go
pir/pircl/shard_cl.go
pir/pircl/shard_cl_test.go
pir/pircpu/shard_cpu.go
pir/pircuda/context_cuda.go
pir/pircuda/shard_cuda.go
pir/pircuda/shard_cuda_test.go
pir/pirtest/shard_tests.go
go vet ./...
# github.com/privacylab/talek/server
server/frontend.go:273:54: conversion from int to string yields a string of one rune, not a string of digits (did you mean fmt.Sprint(x)?)
server/frontend_test.go:18:37: conversion from uint64 to string yields a string of one rune, not a string of digits (did you mean fmt.Sprint(x)?)
server/frontend_test.go:25:36: conversion from int to string yields a string of one rune, not a string of digits (did you mean fmt.Sprint(x)?)
# github.com/privacylab/talek/benchmark/load
benchmark/load/main.go:76:39: conversion from int to string yields a string of one rune, not a string of digits (did you mean fmt.Sprint(x)?)
benchmark/load/main.go:80:57: conversion from int to string yields a string of one rune, not a string of digits (did you mean fmt.Sprint(x)?)
```

--- a/benchmark/load/main.go
+++ b/benchmark/load/main.go
@@ -73,11 +73,11 @@ func main() {
 	clients := make([]*libtalek.Client, *numClients)
 	clientLeaderSock := common.NewFrontendRPC("c0->f0", trustDomainFE.Address)
 	for i := 0; i < *numClients; i++ {
-		clients[i] = libtalek.NewClient("c"+string(i), clientConfig, clientLeaderSock)
+		clients[i] = libtalek.NewClient("c"+fmt.Sprintf("%d", i), clientConfig, clientLeaderSock)
 		handle, _ := libtalek.NewTopic()
 		var origHandle libtalek.Handle
 		origHandle = handle.Handle
-		clients[i].Publish(handle, []byte("Hello from client"+string(i)))
+		clients[i].Publish(handle, []byte("Hello from client"+fmt.Sprintf("%d", i)))
 		fmt.Printf("Published. Waiting for response.")
 		data := clients[i].Poll(&origHandle)
 		_ = <-data

--- a/server/frontend.go
+++ b/server/frontend.go
@@ -6,6 +6,7 @@ import (
 	"crypto/sha256"
 	"encoding/binary"
 	"errors"
+	"fmt"
 	"log"
 	"os"
 	"sync/atomic"
@@ -270,7 +271,7 @@ func (fe *Frontend) triggerBatchRead(batch []*readRequest) error {
 			fe.log.Fatalf("Error making read to replica %d: %v%v", i, err, replies[i].Err)
 		}
 		if len(replies[i].Replies) != len(batch) {
-			replicaErr = errors.New("failure from Replica " + string(i))
+			replicaErr = errors.New("failure from Replica " + fmt.Sprintf("%d", i))
 			fe.log.Fatalf("Replica %d gave the wrong number of replies (%d instead of %d)", i, len(replies[i].Replies), len(batch))
 		}
 	}

--- a/server/frontend_test.go
+++ b/server/frontend_test.go
@@ -1,6 +1,7 @@
 package server
 
 import (
+	"fmt"
 	"testing"
 	"time"
 
@@ -15,14 +16,14 @@ func (m *mockReplica) Write(args *common.ReplicaWriteArgs, reply *common.Replica
 	if m.calls == nil {
 		m.calls = make([]string, 1)
 	}
-	m.calls = append(m.calls, "write-"+string(args.GlobalSeqNo))
+	m.calls = append(m.calls, "write-"+fmt.Sprintf("%d", args.GlobalSeqNo))
 	return nil
 }
 func (m *mockReplica) BatchRead(args *common.BatchReadRequest, reply *common.BatchReadReply) error {
 	if m.calls == nil {
 		m.calls = make([]string, 1)
 	}
-	m.calls = append(m.calls, "read-"+string(len(args.Args)))
+	m.calls = append(m.calls, "read-"+fmt.Sprintf("%d", (len(args.Args))))
 	reply.Replies = make([]common.ReadReply, len(args.Args))
 	return nil
 }

